### PR TITLE
Add Webpack Bundle Analyzer to config

### PR DESCRIPTION
**Pull request by andyg0808**
This allows us to see what's making up our various bundles. We're hunting for what might be leading to slow boot times on our application.

### CODE REVIEW

File: frontend/next.config.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ const withTM = require('next-transpile-modules')([
 
 const { withSentryConfig } = require('@sentry/nextjs');
 
+const withBundleAnalyzer =
+   process.env !== 'production'
```
Comment (danielbeardsley) [inline frontend/next.config.js:16]: `process.env` is an object here.

Comment (andyg0808) [reply frontend/next.config.js]: Fixed in [f1cd17f](https://github.com/iFixit/react-commerce/pull/347/commits/f1cd17f2e668cebed017dfac326ae1add49137cb)

**Comment by danielbeardsley** [comment]
CR :+1:

**Review by danielbeardsley — APPROVED** [review]
CR :+1:

**Comment by andyg0808** [comment]
QA :+1: 
This generated the analysis as desired:
image

**Pull request merged**